docs/src/markdownDocs/themes/customization.mdx

```diff
@@ -8,7 +8,7 @@ Whether you are developing a white-labeled product or need to modify a component
 
 # Changing Individual Component Styles
 
-If you want to change Brightlayer UI component styles on a case-by-case basis, please refer to our [style overrides guide](https://github.com/etn-ccis/blui-react-component-library/blob/master/docs/src/shared/markdown/StyleOverridesGuide.md). Note, however, that this approach may drive up maintenance costs significantly in the long run, especially if you have to maintain multiple products. The better approach in this scenario is to build a custom theme.
+If you want to change Brightlayer UI component styles on a case-by-case basis, please refer to our [style overrides guide](https://github.com/etn-ccis/blui-react-component-library/blob/master/docs/src/shared/markdown/StyleOverridesGuide.md). However, if you need to sweep color changes throughout the entire application, it would be more effective to override Brightlayer UI's themes or build your own custom theme.
```

Comment:
I think Joe is using this definition of sweep — "to quickly spread through and influence an area", but I suppose if it is causing confusions to readers, we should use strictly simple English

Reply:
```suggestion
If you want to change Brightlayer UI component styles on a case-by-case basis, please refer to our [style overrides guide](https://github.com/etn-ccis/blui-react-component-library/blob/master/docs/src/shared/markdown/StyleOverridesGuide.md). However, if you need to apply consistent changes throughout the entire application, such as replacing all theme colors with your own, it would be more effective to override Brightlayer UI's themes or build your own custom theme.
```